.github/workflows/pr.yaml

```diff
@@ -30,6 +30,7 @@ jobs:
     uses: rapidsai/shared-action-workflows/.github/workflows/checks.yaml@branch-23.10
     with:
       enable_check_generated_files: false
+      enable_check_pr_job_dependencies: false
```

Comment:
Is this so that the `devcontainers` step always runs or what is the motivation for this?